arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/call/CallSection.java

```diff
@@ -224,13 +220,13 @@ public CallSection(Hub hub, MessageFrame frame) {
     hub.defers().scheduleForPostTransaction(this);
 
     // The CALL is now unexceptional and un-aborted
-    refineUndefinedScenario(hub);
-    CallScenarioFragment.CallScenario scenario = scenarioFragment.getScenario();
+    refineUndefinedScenario(hub, frame);
+    final CallScenarioFragment.CallScenario scenario = scenarioFragment.getScenario();
     switch (scenario) {
-      case CALL_ABORT_WONT_REVERT -> abortingCall(hub);
       case CALL_EOA_UNDEFINED -> eoaProcessing(hub);
-      case CALL_PRC_UNDEFINED -> prcProcessing(hub);
       case CALL_SMC_UNDEFINED -> smcProcessing(hub, frame);
+      case CALL_PRC_UNDEFINED -> prcProcessing(hub);
+      case CALL_ABORT_WONT_REVERT -> abortingCall(hub);
```

Comment:
Is this reorg an optimization à la "most CALL's will be EOA / SMC and aborts will be rare" ?

Reply:
yes, only.